src/parquet/schema/descriptor.h

```diff
@@ -99,14 +99,18 @@ class SchemaDescriptor {
 
   const schema::NodePtr& schema() const { return schema_; }
 
+  const schema::GroupNode* group() const { return group_; }
+
+  const schema::NodePtr& base(int i) const;
```

Comment:
Add a comment explaining this method -- I'm wondering what might be a more descriptive name, like `GetLeafRoot` or something